mock/store.go

```diff
@@ -25,6 +27,17 @@ type RulesetService struct {
 	EvalVersionFn    func(ctx context.Context, path, version string, params rule.Params) (*regula.EvalResult, error)
 }
 
+// Get runs GetFn if provided and increments GetCount when invoked.
```

Comment:
Somewhere (either here, or on the structure definition) we should document what `GetFn` is supposed to get - it might seem obvious from the context, but if we're allowing someone to provide this function, they should know what it is expected to do.